ckanext/iati/emailer.py

```diff
@@ -90,4 +90,20 @@ def send_email(content, subject, to, from_=FROM, content_type="plain"):
 
 Kind regards,<br>
 IATI Technical Team<br>
-'''
\ No newline at end of file
+'''
+
+data_not_xml_email_body = '''
+Dear {user_name},<br><br>
+
+You're receiving this email because you are the admin user of {publisher_name}'s IATI Registry account.<br>
+
+We've detected an issue with the following IATI dataset in your IATI Registry account: {publisher_registry_dataset_link}<br>
+
+(When the file is not XML) The dataset is not in a valid IATI XML format. Please see our <a href="https://iatistandard.org/en/guidance/publishing-data/creating-files/">Guidance pages</a> on how to publish your data in IATI XML format.<br>
```

Comment:
Is this a meta direction where we're supposed to be including this only if the file is not xml?

Reply:
yup, split them out